### ISSUE TITLE
Containers seem hung; "docker rm -f" hangs

### ISSUE THREAD

**Issue by onmishkin** [docker/for-mac#1607]
Services running in my containers appear non-responsive. "docker rm -f ..." hangs on all my containers. FWIW "docker ps" works fine. Nothing interesting in the Console log. Happens once every day or so. (Things working before the end of the work day then not working at the beginning of the next work day. Machine was _not_ put in sleep mode or disconnected from network in between.) The only solution I've found is to stop and restart Docker.

Diag ID: FF7D4587-F19D-49D1-96B0-EBC7133A7A18 (not sure if the diag tool actually succeeded in uploading data though).

**Comment by yallop** [comment]
Thanks for the report, @onmishkin.  The logs indicate a couple of problems, one with a kernel crash (similar to #87), and one file system issue.

It looks like you're running a rather old version of Docker for Mac (1.12.1), and we believe that these issues may be fixed in more recent versions.  Could you try upgrading to see whether that fixes the problem for you?

**Comment by yallop** [comment]
@onmishkin: the only logs associated with your diagnostic ID appear to be rather old, which probably explains the version discrepancy.  Could you give running the diagnostic tool and uploading another try (outside firewalls, etc., if possible)?

**Comment by onmishkin** [comment]
@yallop : I just ran the diag tool again. It reports an ID of FF7D4587-F19D-49D1-96B0-EBC7133A7A18 and AFAICT it uploaded the data (although I'm saying this solely on the basis of the tool not having indicated any error).

Note that after I originally opened this issue yesterday I restarted Docker. It's been running OK since then but if history is a guide I'll see the problem again within a day or so. I will run the diag tool again when that happens.

Thanks.
